dll/ntdll/ldr/ldrutils.c

```diff
@@ -2455,6 +2455,7 @@ LdrpLoadDll(IN BOOLEAN Redirected,
     RtlCopyUnicodeString(&RawDllName, DllName);
 
     /* Find the extension, if present */
+    /* NOTE: Access violation is expected here in some cases (Buffer[-1]) */
```

Comment:
Since it appears that this PR doesn't fix anything per your own tests, I would suggest:
- keep this added NOTE here, but revert everything else in this file;
- and keep the test, so that we get some testing for LdrLoadDll().

Reply:
Reverted except this comment.